app/controllers/albums

```diff
@@ -0,0 +1,27 @@
+const rp = require('request-promise');
+const typicodePath = 'https://jsonplaceholder.typicode.com';
+
+const options = {
+    uri: '',
+    json: true, // Automatically parses the JSON string in the response
+    resolveWithFullResponse: false
+};
+
+const albums = (req, res, next) => {
+    options.uri = `${typicodePath}/albums`;
+    rp(options)
+    .then(json =>{
+        res.status(200).send(json);
+    });
```

Comment:
What happens if the albums server can't respond? You should add a catch to handle service errors. Try to have the catch forward an error like the ones on errors.js, made for service failures.